index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awsome Books with ES6</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <section class="header--nav--and--title">
+          <h1>Awesome Books</h1>
+          <nav>
+            <ul class="link-container">
+              <li class="list nav--link">List</li>
+              <li class="add nav--link">Add items</li>
+              <li class="contact nav--link">Contact</li>
+            </ul>
+          </nav>
+        </section>
+        <div class="time--and--date">
+          <p class="date--para">
+          </p>
+        </div>
+      </header>
+      <main>
+        <section id="list" class="list--container containers">
+          <h2>All Awesome Books</h2>
+          <ul class="book-list">
+        </section>
+        <section id="contact" class="contact--info containers">
+          <h2>Contact information</h2>
+          <p>Do you have any question or you just want to say "Hello"?</p>
+          <p>You can reach out to us!</p>
+          <ul>
+            <li>Our e-mail: Nelofarzabi2000@gmail.com</li>
+            <li>Our phone number: +93789342113</li>
+            <li>Our address: second streed of khair kahna , Kabul Afg </li>
+          </ul>
+        </section>
+        <form id="add" class="form containers">
+          <h2 class="h2">Add a new book</h2><br>
+          <div>
+            <input type="text" class="title" placeholder="Title"><br><br>
+            <input type="text" class="author" placeholder="Author"><br><br>
+            <button class="add--btn" type="submit">Add</button>
+          </div>
+       </form>
```

Comment:
- [ ] [OPTIONAL] Your form adds objects correctly but I think resetting the form after submission would be great for user experience. That way, they don't make the mistake of adding the same entries twice or having to clear everything before writing :thinking: